README.md

```diff
@@ -85,6 +85,13 @@ vbatts@valse ~/src/vb/git-validation (master) $ echo $?
 1
 ```
 
+Excluding paths that are out of the scope of your project:
+```bash
```

Comment:
This isn't Bash, it's a snippet of an interactive Bash session.  If you want to highlight it, I recommend [`console`][1].

[1]: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4090